### DISCUSSION TITLE
No cache on helm pull

### DISCUSSION THREAD

**Discussion by jycamier** [helmfile/helmfile#227] [Ideas]
Today, the charts are pulled in a temporary directory each time a release is declared.
When we create many releases from the same base chart, this one is pulled for each release and :
- the deployment of a full cluster is longer
- the OCI registry could have rate limits

A cache directory may very interesting.

**Comment by yxxhero** [comment]
@jycamier do you have idea for this. PR is welcome.

**Comment by olivierboudet** [comment]
I am also interested on this, I will look if possible to do a PR.

**Reply by yxxhero** [reply]
Cool
